jechova.py

```diff
@@ -41,9 +45,13 @@
 
     channel = '#' + args.channel.lstrip('#')
     client = WebClient(token=os.environ['SLACK_API_TOKEN'])
-    client.chat_postMessage(channel=channel, text=dedent('''
-        Máte téma? A mohla bych ho vidět? :eyes:\n\n
-        Pokud už tušíte, o čem Pyvo bude, přidejte to na pyvo.cz — může
-        to udělat kdokoliv, v 5 jednoduchých krocích:
-        https://docs.pyvec.org/guides/meetup.html#informace-na-pyvo-cz
-    ''').strip())
+
+    docs_url = "https://docs.pyvec.org/guides/meetup.html#informace-na-pyvo-cz"
+    web_url = f'https://pyvo.cz/{args.pyvo_slug}/'
+    client.chat_postMessage(channel=channel, text=' '.join((
+        'Máte téma? A mohla bych ho vidět? :eyes:\n',
+        'Pokud už tušíte, o čem Pyvo bude, přidejte to na pyvo.cz',
+        'přes odkaz _přidat informace o dalším srazu_ na',
+        f'<{web_url}|pyvo.cz/{args.pyvo_slug}>.\n',
+        '(Složitější případy řešte v <#C97Q9HHHT>)',
```

Comment:
Testoval jsi, jestli to funguje, jo? Myslím ty všelijaké formátovací značky a tak. Budu ti věřit :) Tady bych možná jen za řádek doplnil komentář, že `#C97Q9HHHT` je v lidské mluvě (předpokládám) `#pyvo`:

```suggestion
        '(Složitější případy řešte v <#C97Q9HHHT>)',  # makes a link to the #pyvo channel
```

Reply:
Funguje. Naopak `#pyvo` nefunguje :)